src/LlamaPayV2Payer.sol

```diff
@@ -494,7 +511,23 @@ contract LlamaPayV2Payer is ERC721, BoringBatchable {
         /// Update Token info to get last update
         Stream storage stream = streams[_id];
         _updateToken(stream.token);
-        uint48 lastUpdate = tokens[stream.token].lastUpdate;
+        Token storage token = tokens[stream.token];
+
+        unchecked {
```

Comment:
this is very unlikely so you could just build a new function so ppl can claim their debt, that way you dont need to add an extra sload to every call